test/sets.jl

```diff
@@ -734,7 +734,6 @@ end
     x = @inferred replace([1, 2], 2=>missing)
     @test isequal(x, [1, missing]) && x isa Vector{Union{Int, Missing}}
 
-    @test_broken @inferred replace([1, missing], missing=>2)
```

Comment:
This should be
```julia
@test @inferred replace([1, missing], missing=>2) == [1, 2]
```
?

Reply:
That happens two lines below.  The `@test_broken` check was added explicitly to check that inference itself fails here, independent of checking whether the value is correct.

Reply:
I guess the better fix is to keep the `@inferred` on the next line though.